new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItJrfDomainInPV.java

```diff
@@ -65,38 +110,31 @@
    * @param namespaces injected by JUnit
    */
   @BeforeAll
-  public static void initAll(@Namespaces(1) List<String> namespaces) {
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
```

Comment:
Now you need to initialize the logger in this method and remove the implements LoggedTest.

See https://github.com/oracle/weblogic-kubernetes-operator/pull/1765

Reply:
Yeah, when I merged to the latest develop branch I can see the conflicts.

Reply:
Fixed.